hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/upgrade/HDDSLayoutFeature.java

```diff
@@ -41,7 +41,9 @@ public enum HDDSLayoutFeature implements LayoutFeature {
   WEBUI_PORTS_IN_DATANODEDETAILS(6, "Adding HTTP and HTTPS ports " +
       "to DatanodeDetails."),
   HADOOP_PRC_PORTS_IN_DATANODEDETAILS(7, "Adding Hadoop RPC ports " +
-                                     "to DatanodeDetails.");
+                                     "to DatanodeDetails."),
+  LAST_CHUNK_TABLE(8, "Datanode RocksDB Schema Version 3 has an extra table " +
```

Comment:
How about change the feature name to HBASE_SUPPORT? For file recovery, a new table finalize_blocks  is also introduced. So for HBASE, as of now, two new tables are introduced for it. So if use the name HBASE_SUPPORT, we can cover these two new tables, and also the potential new changes in later.

Reply:
Ok